### PR TITLE
Stop voice broadcast recording on redaction

### PR DESCRIPTION
**Pull request by weeman1337**
Part of https://github.com/vector-im/element-web/issues/23282

## Checklist

* [x] Tests written for new code (and old code if feasible)
* [x] Linter and other CI checks pass
* [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: test/voice-broadcast/stores/VoiceBroadcastRecordingsStore-test.ts [hunk 1]
```diff
@@ -110,6 +99,31 @@ describe("VoiceBroadcastRecordingsStore", () => {
             it("should emit a current changed event", () => {
                 expect(onCurrentChanged).toHaveBeenCalledWith(null);
             });
+
+            it("and calling it again should work", () => {
+                recordings.clearCurrent();
```
Comment (SimonBrandner) [inline test/voice-broadcast/stores/VoiceBroadcastRecordingsStore-test.ts:104]: This seems to be missing an `expect` statement?

Comment (weeman1337) [reply test/voice-broadcast/stores/VoiceBroadcastRecordingsStore-test.ts:104]: It should just not explode :laughing:

Comment (SimonBrandner) [reply test/voice-broadcast/stores/VoiceBroadcastRecordingsStore-test.ts:104]: A comment then please?

**Review by kerryarchibald — APPROVED** [review]
LGTM
Tested on netlify, works as described

**Review by SimonBrandner — APPROVED** [review]

**Pull request merged**